xsd/netex_part_1/part1_networkDescription/netex_routeInstruction_version.xsd

```diff
@@ -73,11 +73,18 @@ Rail transport, Roads and Road transport
 		<xsd:complexContent>
 			<xsd:extension base="strictContainmentAggregationStructure">
 				<xsd:sequence>
-					<xsd:element ref="RouteInstruction" maxOccurs="unbounded">
-						<xsd:annotation>
-							<xsd:documentation>An Instruction on how to follow a ROUTE through the network.</xsd:documentation>
-						</xsd:annotation>
-					</xsd:element>
+					<xsd:choice minOccurs="0">
```

Comment:
This is not what I like. Because this thing is part of a sequence, you can stil do:

RouteInstructionRef
RouteInstruction
RouteInstructionRef
RouteInstruction

In additition it does **not** prevent an empty routeInstructions_RelStructure.

Reply:
ok done again.